src/test/java/api/requests/PickSlipsTests.java

```diff
@@ -35,326 +40,357 @@
-
-    final UUID servicePointId = servicePointsFixture.cd1().getId();
-
-    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+  public void responseIsEmptyWhenItemHasOpenPageRequestWithWrongStatus() {
+    UUID servicePointId = servicePointsFixture.cd1().getId();
+    InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
 
     requestsFixture.place(new RequestBuilder()
-        .page()
-        .withStatus(RequestStatus.OPEN_AWAITING_PICKUP.getValue())
-        .withPickupServicePointId(servicePointId)
-        .forItem(smallAngryPlanet)
-        .by(usersFixture.james()));
+      .page()
+      .withStatus(RequestStatus.OPEN_AWAITING_PICKUP.getValue())
+      .withPickupServicePointId(servicePointId)
+      .forItem(smallAngryPlanet)
+      .by(usersFixture.james()));
 
     Response response = ResourceClient.forPickSlips().getById(servicePointId);
-    assertThat(response.getStatusCode(), is(HTTP_OK));
 
+    assertThat(response.getStatusCode(), is(HTTP_OK));
     assertResponseHasItems(response, 0);
   }
 
   @Test
-  public void reportContainsPagedItemWithOpenUnfilledRequest() {
-
-    UUID servicePointId = servicePointsFixture.cd1().getId();
-    InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
-
-    requestsFixture.place(new RequestBuilder()
-        .withStatus(RequestStatus.OPEN_NOT_YET_FILLED.getValue())
-        .page()
-        .withPickupServicePointId(servicePointId)
-        .forItem(smallAngryPlanet)
-        .by(usersFixture.james()));
+  public void responseContainsAllAvailableTokens() {
+    IndividualResource servicePoint = servicePointsFixture.cd1();
+    UUID servicePointId = servicePoint.getId();
+    IndividualResource locationResource = locationsFixture.thirdFloor();
+    IndividualResource addressTypeResource = addressTypesFixture.home();
+    Address address = AddressExamples.mainStreet();
+    IndividualResource requesterResource = usersFixture.steve(builder -> builder.withAddress(address));
+    DateTime requestDate = new DateTime(2019, 7, 22, 10, 22, 54, DateTimeZone.UTC);
+    LocalDate requestExpiration = new LocalDate(2019, 7, 30);
+    LocalDate holdShelfExpiration = new LocalDate(2019, 8, 31);
+    IndividualResource materialTypeResource = materialTypesFixture.book();
+    IndividualResource loanTypeResource = loanTypesFixture.canCirculate();
+
+    InventoryItemResource itemResource = itemsFixture.basedUponSmallAngryPlanet(builder ->
+      builder.withEnumeration("v.70:no.7-12")
+        .withVolume("vol.1")
+        .withChronology("1984:July-Dec.")
+        .withYearCaption(Arrays.asList("1984", "1985"))
+        .withCopyNumber("cp.2")
+        .withNumberOfPieces("3")
+        .withDescriptionOfPieces("Description of three pieces")
+        .withPermanentLocation(locationResource)
+        .withMaterialType(materialTypeResource.getId())
+        .withPermanentLoanType(loanTypeResource.getId())
+    );
```

Comment:
Personally, I would include this on the preceding line. What benefit is there to it being on a separate line?

Reply:
No problem. It just separates the body of the lambda from enclosing method invocation.
These `)))))))` drive me crazy :)

Reply:
Yeah, the nested brackets are annoying